nrf52840-hal/src/ieee802154.rs

```diff
@@ -38,11 +35,26 @@ pub const DEFAULT_TXPOWER: TxPower = TxPower::_0dBm;
 /// Default Start of Frame Delimiter = `0xA7` (IEEE compliant)
 pub const DEFAULT_SFD: u8 = 0xA7;
 
+/// Energy measurements above this value mean that the channel is assumed to be busy.
+/// Note the the measurement range is 0..0xFF - where 0 means that the received power was
+/// less than 10 dB above the selected receiver sensitivity. This does not correlate to dBm,
+/// see the nrf52840 Product Specification for conversion instructions.
+pub const DEFAULT_ED_THRESHOLD: u8 = 0x50; // note: picked trough trial & error
+
+/// Default Clear Channel Assessment method = Carrier sense
+pub const DEFAULT_CCA: Cca = Cca::EnergyDetection {ed_threshold: DEFAULT_ED_THRESHOLD}; // todo undo
+
+/// based on CSMA-CA macMaxBe from 802.15.4-15, since -2006 isn't freely available to find out
+/// the default number of retries
+pub const NUM_SEND_RETRIES: u8 = 5;
+
 // TODO expose the other variants in `pac::CCAMODE_A`
 /// Clear Channel Assessment method
 pub enum Cca {
     /// Carrier sense
     CarrierSense,
+    /// Energy above the Energy Detection threshold
+    EnergyDetection { ed_threshold: u8 } ,
```

Comment:
LGTM. I would a note / warning saying that using a bad `ed_threshold` value (too low or too high) may cause CCA to always fail and `send` to block forever.